core/src/test/scala/unit/kafka/log/remote/RemoteLogManagerTest.scala

```diff
@@ -273,4 +274,4 @@ class RemoteLogManagerTest {
     new RemoteLogManagerConfig(config)
   }
 
-}
\ No newline at end of file
+}
```

Comment:
extra newline

Reply:
@junrao I think most editors expect a new line at the end. For example, IntelliJ adds it automatically after I deleted it. And vim doesn't even show that there is a new line. So, I think we should probably leave it in.

Reply:
Thanks for the explanation. Sounds good.